content/klc/S6.2.adoc

```diff
@@ -1,9 +1,24 @@
 +++
-brief = "Component fields must be filled appropriately"
+brief = "Symbol and alias fields and metadata filled out as required"
 +++
 
 . *Reference* field is selected appropriately for the symbol and is _visible_
 . *Value* field contains the name of the symbol and is _visible_
-. *Footprint* field contains footprint link for atomic parts, and is empty for generic parts. It must be _invisible_
-. *Datasheet* field is left empty, and is _invisible_
-. The symbol contains no other custom fields
\ No newline at end of file
+. *Footprint* field contains footprint link for fully specified symbols, and is empty for generic symbols. It must be _invisible_
+. *Datasheet* entry [1] is filled out, and is _invisible_
```

Comment:
This style of using a numbered reference in brackets is not used elsewhere in KLC that I can find. I see a a separate section below without any linking or sub-bullets. Do you want to introduce this new style?

Reply:
This is a footnote for adding additional information. We do not have a footnote anywhere else in the KLC (it was not necessary to clarify things like that in other rules. As this is really one of the few places where we need to explain strange KiCad behaviour)

Reply:
It's not clear to me why this needs to be done this way instead of just being underneath the `Datasheet` section. But OK.

Reply:
I wanted to keep this rule short so that users read it. What is below basically gives additional clarification for users who are still interested. 

I learned with bad experiences that users turn their brain off after a few sentences so the most important info should be kept short. Adding additional stuff for the few users that are interested is still worth it. Especially if we ever get a discussion in a pull request because then we can point to this additional information.